spec/pluralrules.html

```diff
@@ -146,23 +139,13 @@
         1. Assert: Type(_n_) is Number.
         1. If isFinite(_n_) is *false*, then
           1. Return *"other"*.
-        1. If the _pluralRules_.[[MinimumSignificantDigits]] is not *undefined* and _pluralRules_.[[MaximumSignificantDigits]] is not *undefined*, then
-          1. Let _s_ be ToRawPrecision(_n_, _pluralRules_.[[MinimumSignificantDigits]], _pluralRules_.[[MaximumSignificantDigits]]).
-        1. Else,
-          1. Let _s_ be ToRawFixed(_n_, _pluralRules_.[[MinimumIntegerDigits]], _pluralRules_.[[MinimumFractionDigits]], _pluralRules_.[[MaximumFractionDigits]]).
+        1. Let _s_ be ? FormatNumber(_pluralRules_[[NumberFormat]], _n_).
         1. Let _locale_ be _pluralRules_.[[Locale]].
         1. Let _type_ be _pluralRules_.[[Type]].
         1. Let _operands_ be ? _GetOperands_(_s_).
         1. Return ? PluralRuleSelection(_locale_, _type_, _n_, _operands_).
       </emu-alg>
 
-      <emu-note>
-        The computations rely on Number values and locations within _n_ that are dependent upon the implementation and the effective locale of _pluralRules_ (“ILD") or upon the implementation. The ILD String mentioned, must not contain any characters in the General Category “Number, decimal digit" as specified by the Unicode Standard.
-      </emu-note>
-
-      <emu-note>
-        It is recommended that implementations use the locale data provided by the Common Locale Data Repository (available at <a href="http://cldr.unicode.org/">http://cldr.unicode.org/</a>).
-      </emu-note>
```

Comment:
If this is going to use a new abstract operation `FormatNumberString`, there is not need to have this here, but there.